agent/cmd/start.go

```diff
@@ -156,6 +157,14 @@ func newStartCommand() *cobra.Command {
 				cfg.Subscriptions = viper.GetStringSlice(flagSubscriptions)
 			}
 
+			// Get a single or a list of statsd event handlers
+			handlers := viper.GetString(flagStatsdEventHandlers)
+			if handlers != "" {
+				cfg.StatsdServer.Handlers = splitAndTrim(handlers)
+			} else {
+				cfg.StatsdServer.Handlers = viper.GetStringSlice(flagStatsdEventHandlers)
```

Comment:
This else clause of the conditional is meant to use the default value for the flag if the user didn't pass in any value (or didn't use the flag at all), correct?

Reply:
We could actually replace that whole block (160 to 166) with this single line (165) by defining the flag `flagStatsdEventHandlers` flag as a `StringSlice` instead of a `String` on line  [262](https://github.com/sensu/sensu-go/pull/1288/files#diff-10c3ef0457ad91a0dcfe4a428511a129R262), just like we did for `flagBackendURL`.

Reply:
But then a user would have to use the flag multiple times rather than a comma delimited list, correct? For example: `sensu-agent --statsd-event-handler handler1 --statsd-event-handler handler2` vs. `sensu-agent --statsd-event-handlers handler1,handler2`

Reply:
@nikkiattea Both works with `StringSlice`. For example, if I print the value of `viper.GetStringSlice(flagStatsdEventHandlers)` and its lenght:

```
$ sensu-agent
value: []
len: 0

$ sensu-agent --statsd-event-handlers foo
value: [foo]
len: 1

$ sensu-agent --statsd-event-handlers foo,bar
value: [foo bar]
len: 2

$ sensu-agent --statsd-event-handlers foo --statsd-event-handlers bar
value: [foo bar]
len: 2
```